src/server/api/root.ts

```diff
@@ -15,7 +15,7 @@ export const appRouter = createTRPCRouter({
   application: applicationRouter,
   auth: authRouter,
   preregistration: preregistrationRouter,
-  review: reviewRouter,
```

Comment:
don't replace the review router (I'm guessing this happened when resolving conflicts), either don't create a new router and add ur mutation to the existing reviewRouter (I'd recommend this), or create a new router here for ur reviewRequestRouter

Reply:
this is what's causing build errors lol